backend/src/controllers/v2/environmentController.ts

```diff
@@ -124,7 +161,7 @@ export const renameWorkspaceEnvironment = async (
   if (envIndex === -1) {
     throw new Error("Invalid environment given");
```

Comment:
throw new BadRequestError

Reply:
This one actually isn't in code that I modified, but I switched all the ones I wrote to use a BadRequestError